docs/resources/deployment.md

```diff
@@ -75,8 +75,14 @@ resource "prefect_deployment" "deployment" {
       type               = "git_clone"
       repository         = "https://github.com/some/repo"
       branch             = "main"
-      access_token       = "123abc"
       include_submodules = true
+
+      # For private repositiroeis, choose from either:
```

Comment:
```suggestion
      # For private repositories, choose from either:
```

Reply:
what did you not know about repositiroeis

Reply:
😆